src/server.tsx

```diff
@@ -26,23 +26,27 @@ const DEV = !PROD;
 const template = fs.readFileSync('./dist/template2.html', 'utf8');
 
 const renderSite = async (url: string) => {
+  const asyncContext = createAsyncContext();
   const metadata = getPageMetadata(url);
   const title = metadata.title;
   const description = metadata.description;
-  const asyncContext = createAsyncContext();
+  const context = {};
   const app = (
     <AsyncComponentProvider asyncContext={asyncContext}>
       <StaticRouter
         location={url}
-        context={{}}
+        context={context}
       >
         <App />
       </StaticRouter>
     </AsyncComponentProvider>
   );
   await asyncBootstrapper(app);
   const reactHtml = ReactDOMServer.renderToStaticMarkup(app);
-  return Mustache.render(template, { title, reactHtml, description });
+  return ({
+    is404: 'status' in context ? true : false,
```

Comment:
huuum, we could make this work for any http code easily.
Replace this with:
return {{
  status: context.status